guides/source/getting_started.textile

```diff
@@ -110,7 +110,7 @@ Just add it to your apps Gemfile
 
 <ruby>
 # Gemfile
-gem 'alchemy-devise', github: 'AlchemyCMS/alchemy-devise', branch: '3.5-stable'
+gem 'alchemy-devise', github: 'AlchemyCMS/alchemy-devise'
```

Comment:
That way we advice users to use the latest GitHub `master`. This is not what we want.

Instead we should tell users to use the latest gem release

```suggestion
gem 'alchemy-devise'
```